lib/rails_admin/config/model.rb

```diff
@@ -48,9 +48,7 @@ def initialize(entity)
       end
 
       def excluded?
-        return @excluded if defined?(@excluded)
-
-        @excluded = !RailsAdmin::AbstractModel.all.collect(&:model_name).include?(abstract_model.try(:model_name))
+        @excluded ||= !RailsAdmin::AbstractModel.all.collect(&:model_name).include?(abstract_model.try(:model_name))
```

Comment:
Is the behaviour change here intended?  This change removes the benefit of memoisation when the result is falsely.

Ref: http://gavinmiller.io/2013/advanced-memoization-in-ruby/

Reply:
Good catch. No that is not intended. If you put up a PR I'll approve, otherwise I'll try to get to it later today.

Reply:
Restored in #3587. Thanks for reporting.